homeassistant/components/climate/const.py

```diff
@@ -56,6 +56,9 @@
 # Device is reacting to activity (e.g. movement sensors)
 PRESET_ACTIVITY = 'activity'
 
+# Device is in manufacturer specific mode (e.g. setting the valve manually)
+PRESET_MANUFACTURER_SPECIFIC = 'Manufacturer Specific'
```

Comment:
Move that to zwave because no other platform use that

Reply:
What so you think about removing this completey? Since all unmatched modes will be used as presets this should not break anything.

Reply:
It's your PR :)

Reply:
I've moved this to zwave